core/src/main/java/org/infinispan/protostream/ProtobufUtil.java

```diff
@@ -142,6 +175,8 @@ public static ByteBuffer toWrappedByteBuffer(ImmutableSerializationContext ctx,
 
       boolean isFirstField = true;
 
+      boolean isTypeMetaField = false;
```

Comment:
I don't think we need ```isTypeMetaField```. The logic for writing the _type field only once and adding a proper coma after it can be also done using ```isFirstField```